### ISSUE TITLE
Small problem with translated text coming from Module...

### ISSUE THREAD

**Issue by jdriesen** [leonardovilarinho/vue-multilanguage#6]
Hi Leonardo,

I have one last small question...
I can't make it work to pick up a translation from a component...

I've made a very small demo project, based on your demo.
I've added a component named 'myfooter'.

Can I kindly ask to take a look a the sourcecode of the myfooter component ?
I've added a detailed comment describing my problem.

[https://github.com/jdriesen/forleonardo](https://github.com/jdriesen/forleonardo)

Thanks in advance for your time, Leonardo.

Kind regards,
Johnny

**Comment by leonardovilarinho** [comment]
I'm working on it

**Comment by jdriesen** [comment]
Thanks man !
I appreciate your effort !

I really do.

Grtz,
Johnny

**Comment by leonardovilarinho** [comment]
I corrected the problem and sent a pull request.

See the changes in my PR.

To declare parameters in messages use `{paramName}`, and to send parameters, simply send an object in the `translate` method or in the directive. For example:

```
// by method
this.translate(this.$language, 'path.to.msg', {paramName: 'my value'})

// or directive
<div v-lang.path.to.msg="{paramName: 'my value'}"
```

The `path.to.msg` is the index of your message within the object of languages, it would be the same as accessing a `user.name`, however you are accessing there inside the object passed to the plugin.

For example, I have the object:
```
{
    en: {
        footer: {
            description: 'by Brazil'
        },
        hello: 'Hello World'
    },
    pt: {
        footer: {
            description: 'por Brasil'
        },
        hello: 'Olá mundo'
    }
}
```

Accessing `footer.description` would fetch the message`by Brazil`, since `en` is my active language in the system.

Accessing `hello` would get the message`Hello world`.

I hope I have helped to understand, my English is weak, so I can not speak much.
